app/components/index.jsx

```diff
@@ -1,27 +1,29 @@
-var React = require('react'),
-    Router = require('react-router'),
-    DefaultRoute = Router.DefaultRoute,
-    RouteHandler = Router.RouteHandler,
-    Route = Router.Route,
-    Header = require('./components/header.jsx'),
-    Status = require('./components/status.jsx'),
-    Search = require('./pages/search.jsx'),
-    Homepage = require('./pages/homepage.jsx');
+const React = require('react');
+const Router = require('react-router');
+const DefaultRoute = Router.DefaultRoute;
+const RouteHandler = Router.RouteHandler;
+const Route = Router.Route;
+const Header = require('header');
+const Status = require('status');
+const Search = require('pages/search');
+const Homepage = require('pages/homepage');
 
-var App = React.createClass({
+require('./index.css');
```

Comment:
With the css-loader and style-loader it will add some JS to the bundle that will add these styles to the page when the JS is executed. I think there are other loaders and configuration you can use to have it extract it to an external stylesheet file that you add to the page somehow, but this is simpler and will work just fine for Mosaic.